near-sdk-sim/src/user.rs

```diff
@@ -154,18 +154,35 @@ impl UserAccount {
 
     /// Make a contract call.  `pending_tx` includes the receiver, the method to call as well as its arguments.
     /// Note: You will most likely not be using this method directly but rather the [`call!`](./macro.call.html) macro.
-    pub fn call(
+    pub fn function_call(
         &self,
         pending_tx: PendingContractTx,
         deposit: Balance,
         gas: Gas,
     ) -> ExecutionResult {
-        self.submit_transaction(self.transaction(pending_tx.receiver_id).function_call(
-            pending_tx.method.to_string(),
-            pending_tx.args,
-            gas,
-            deposit,
-        ))
+      self.call(pending_tx.receiver_id.clone(),
+          &pending_tx.method,
+          &pending_tx.args,
+          gas,
+          deposit,
+      )
+        
+    }
+
+    pub fn call(
```

Comment:
Heads up: This change is going to break the public API. What is the state of `near-sdk-sim` in terms of API stability? Is this fine to introduce breaking changes?

Reply:
@frol, we are about to publish for the first time so I thought this would be the opportunity to break the API.  As it is written now, you would just need to rename the methods so it's not too bad of a breaking change.